packages/happo-target-firefox/src/__tests__/checkBrowserVersion-test.js

```diff
@@ -44,18 +36,3 @@ describe('when the version string is unrecognized', () => {
         'Failed to parse Firefox version string "Godzilla Firefox 47.0"');
     }));
 });
-
-describe('when using a different driver', () => {
```

Comment:
since this is now the "firefox target", i removed the logic where the driver could be non-firefox, since i think we will end up implementing that as a separate module